### DISCUSSION TITLE
Network multiplayer connectivity

### DISCUSSION THREAD

**Discussion by Nalajalan** [Swordfish90/Lemuroid#855] [Q&A]
Hi everyone, 

I think it would be great to connect several android devices through lan or wan. 

My present idea was to connect two or three GBA together or with a Game Cube to play The legend of Zelda four swords (GBA or GameCube+GBA version).

And that's where I'm little confused.

If I understand, lemuroid (that is really a great emulator) use mGBA but mGBA website claims that there is none Android version of it...
Moreover mGBA let connecting with other emulators like other mGBA instances or dolphin (for Game Cube) but lemuroid doesn't seem to let network multiplayer at all...

**Comment by andjeozla** [comment]
Ok. So Lemuroid isn't using mGBA. It is only using the core, not the full app. Lemuroid is actually an easier to setup RetroArch. You need to learn more about RetroArch in order to understand how Lemuroid works. RetroArch only uses cores of some of the emulators, there are always some exceptions.
